src/utils/menu.js

```diff
@@ -1,52 +1,85 @@
 const menu = [
   {
-    name: 'Apps',
+    name: '1hive Products',
```

Comment:
I wouldn't really call them products, I think "projects" is more appropriate in this context

Reply:
Okay I will change,